paper.tex

```diff
@@ -68,28 +68,62 @@ \section*{Ancestral graphs and stochastic processes}
+Griffiths~\citep{griffiths1991two,griffiths1997ancestral} as a branching-coalescing 
+stochastic process whose realisations are graphs which into which all of the
+marginal ancestral trees are embedded. Coalescences encode common ancestry
+as they do in the coalescent, while branching events encode points at which a
+marginal tree changes from one position along the genome to the next.
+
+There are several formulations of the ARG, which vary by
+the amount of extra information they capture beyond a minimal representation
+of all marginal trees. Two prominent examples are the ``big''
+ARG~\citep{ethier1990two}, which has a particularly simple specification as a
+stochastic process but is computationally infeasible, and the ``little ARG''
+traversed by Hudson's algorithm~\citep{hudson1983properties}, which
+has a more complex state space and dynamics but can be implemented with 
+a more manageable runtime and memory footprint. An efficient simulation
+implementation has become available only recently \citep{kelleher2016efficient},
+prior to which state-of-the-art simulators were based on the so-called sequentially
+Markovian coalescent (SMC) \citep{mcvean2005approximating}, which delivers
+performance gains by disallowing mergers which would result in trapped
+non-ancestral material. An alternative restricted model is
+\texttt{ClonalFrame}  \citep{didelot2007inference}, which fixes one distinguished
+tree and disallows recombination in any other marginal tree.
+
+Similar graph encodings have also been introduced to model other mechanisms
+which change the local ancestral tree along the genome, such as gene
+conversion~\citep{wiuf2000coalescent} and
+horizontal gene transfer~\citep{baumdicker2014infinitely}.
+
+The idea of using branching-coalescing stochastic processes 
+to model genetic ancestral trees has also found application in natural selection.
+The Ancestral Selection Graph (ASG)~\citep{krone1997ancestral,neuhauser1997genealogy}
+uses dynamics identical to the ``big" ARG to simulate an ensemble of correlated
+potential ancestral trees. Weak genic selection is incorporated by sampling a 
+true ancestry from the ensemble in a non-uniform way.
+There is no ASG analogue of the more computationally tractable ``little" ARG,
+though some gains in tractability can be made by considering typed 
+lineages~\citep{etheridge2009coalescent} or by leveraging perfect simulation
+techniques when recurrent mutation is present\citep{fearnhead2001perfect}.
+Extensions of the ASG have been developed to frequency-dependent 
+selection~\citep{neuhauser1999ancestral, gonzalezcasanova2018duality},
+unlinked chromosomes~\citep{fearnhead2003ancestral}, recombining
+loci whereupon branching is due to both selection and
+recombination~\citep{donnelly1999genealogical}, and high fecundity 
+reproduction~\citep{gonzalezcasanova2018duality, koskela2019robust}.
 
 In the theoretical literature, ancestral graphs such as the ARG refer to the
```

Comment:
This is not a sentence modified in this PR, but I think we agreed that in (most of) the theoretical literature, the ARG is assumed to mean *both* the process and the structure generated by it.